packages/experimental/Avatar/src/JSAvatar.types.ts

```diff
@@ -1,10 +1,24 @@
 import type { IViewProps } from '@fluentui-react-native/adapters';
 import { ImageProps, ViewProps, ImageURISource, TextProps, ColorValue } from 'react-native';
 import { IBackgroundColorTokens, IForegroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { Presence } from '../../Badge';
```

Comment:
We should import this from the Badge package instead of using relative pathing, since the package can move in the future.

Reply:
fixed